src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -106,17 +113,22 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
         });
     }
 
+    public void switchMode(ModeEnum mode) {
+        assert mode == null : "Mode should not be null";
+        this.mode = mode;
+    }
+
     /**
      * Fills up all the placeholders of this window.
      */
-    void fillInnerParts() {
```

Comment:
Subject to the discussion above :D

Reply:
I think this is for the initial initialisation of the Ui (to contain the Contacts List as of now). Can be changed later on to show dashboard